macros/utils/table_operations/seed.sql

```diff
@@ -0,0 +1,20 @@
+{% macro seed_elementary_model(relation, csv_path) %}
+  {% do adapter.dispatch('seed_elementary_model', 'elementary')(relation, csv_path) %}
+{% endmacro %}
+
+{% macro default__seed_elementary_model(relation, csv_path) %}
+  {% set table = elementary.get_agate_table().from_csv(csv_path) %}
+  {% do adapter.truncate_relation(relation) %}
```

Comment:
This is a bit dangerous as we can end up with an empty table, can we make it atomic? for example we could create a temp table, insert to this table, then do a drop and rename in one transaction or something like that